pkg/cmd/sponsors/list/list.go

```diff
@@ -0,0 +1,36 @@
+package list
+
+import (
+	"fmt"
+
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+type ListOptions struct {
+	Username string
+	Sponsors []string
+}
+
+func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Command {
+	opts := &ListOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List sponsors for a user",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.Username = args[0]
+			opts.Sponsors = []string{"GitHub"}
+
+			if runF != nil {
+				return runF(opts)
+			}
```

Comment:
As far as I can tell, `runF` is required for testing. What other use-cases might we include a non-nil `runF`?

Reply:
Great question! There's no other reason that I am aware of. This is a bit of test-induced design damage with the primary purpose being as a `spy` for the flag and argument parsing. However, in practice the cost has been low so I haven't given much thought towards how we might address it.